src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -857,10 +875,14 @@ private MethodKind ComputeMethodKind()
                         case WellKnownMemberNames.ImplicitConversionName:
                         case WellKnownMemberNames.ExplicitConversionName:
                             return IsValidUserDefinedOperatorSignature(1) ? MethodKind.Conversion : MethodKind.Ordinary;
-                            // UNDONE: Non-C#-supported overloaded operator case WellKnownMemberNames.ConcatenateOperatorName:
-                            // UNDONE: Non-C#-supported overloaded operator case WellKnownMemberNames.ExponentOperatorName:
-                            // UNDONE: Non-C#-supported overloaded operator case WellKnownMemberNames.IntegerDivisionOperatorName:
-                            // UNDONE: Non-C#-supported overloaded operator case WellKnownMemberNames.LikeOperatorName:
+                        case WellKnownMemberNames.IsOperatorName:
+                            return IsValidUserDefinedOperatorIs() ? MethodKind.UserDefinedOperator : MethodKind.Ordinary;
+                        case WellKnownMemberNames.ConcatenateOperatorName:
+                        case WellKnownMemberNames.ExponentOperatorName:
+                        case WellKnownMemberNames.IntegerDivisionOperatorName:
+                        case WellKnownMemberNames.LikeOperatorName:
```

Comment:
I do not believe there is a value in adding the four cases above, why force compiler to do unnecessary string comparisons. If goal is documentation, I think a comment will be sufficient.